typescript/core/src/v0/ChunkCursor.ts

```diff
@@ -245,7 +244,7 @@ export class ChunkCursor {
           startIndex = sortedIndexBy(
             result.record.records,
             [this.startTime],
-            ([logTime]) => logTime,
+            ([logTime]) => logTime!,
```

Comment:
You might be able to specialize sortedIndexBy <T> value to expect a tuple rather than an array and that would let you drop the `!` if that's interesting to avoid errors in the future with type changes.